topics/scistor.qmd

```diff
@@ -0,0 +1,27 @@
+---
+title: SciStor
+---
```

Comment:
```suggestion
---

```

This is not critical, only doing this so that we can keep all the markdown styling consisten.